src/components/megalinks/components/megalink-multi-image/MegalinkMultiImage.vue

```diff
@@ -219,6 +227,10 @@ export default {
             left: 0;
         }
 
+        .vs-badge {
```

Comment:
To avoid adding this in the megalinks file, can we just add the margin utility class to the badge itself in the slot?

Reply:
Hmm, yes that's probably a better idea, I'd forgotten those were available outside of the library